index.html

```diff
@@ -257,8 +257,47 @@ <h4 class="ab-titles skills"> Skills</h4>
         </div>
       </div>
     </section>
+    <section class="form-container flex-column">
+      <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      <div class="form flex-column">
+        <form class="flex-column " action="https://formspree.io/f/mayzgdra" method="POST" class="form-data">
+          <input type="text" id="name" placeholder="Full name" required maxlength="30">
+          <input type="email" id="email" placeholder="Email address" required>
+          <input type="text" id="message" placeholder="Write something..." required maxlength="500">
```

Comment:
- [ ] Great implementation of your form and its field types. 👍🏿 
However, this `message` form input should have a field `type="textarea"`. Textarea types are more appropriate for more character inputs like messages and paragraphs. 📜